requirements-colbert.txt

```diff
@@ -0,0 +1,4 @@
+colbert-ai=0.2.19
+faiss-gpu
```

Comment:
Is this issue with faiss-gpu fixed or it was not related to the OG ColBERT library?

Reply:
It's not actually fixed, you raise a good point. The only straightforward way to install faiss-gpu with CUDA 12+ is conda or installing from source :/

Reply:
Yeah that's why I raised this. IIRC I made it work with a conda install indeed, but maybe it's more convenient to use source installation to make it compatible with the `pip install -r requirement-colbert.txt` command if that works?

Reply:
It wouldn’t work — installing from source involves building everything yourself. The best option would be to add it to the conda env and make that the preferred way to do things.

Reply:
Oh yeah sorry, I thought it was just installing from the repo but it is a bit more painful.
If the issue is just with faiss-gpu, do we actually need it for small evals or we can go with only cpu?
Otherwise, we can either install it in the base environment or add a flag to the yaml, but the solution I found seems pretty ugly.

Reply:
> If the issue is just with faiss-gpu, do we actually need it for small evals or we can go with only cpu?
> Otherwise, we can either install it in the base environment or add a flag to the yaml, but the solution I found seems pretty ugly.

I'll merge and an issue to do this, but ideally we'd add it to the environment. The smaller datasets are fine on CPU but webis/trec-covid will already be in the 1hr+ processing time on CPU.